qiskit_experiments/analysis/fit_functions.py

```diff
@@ -0,0 +1,75 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+A library of fit functions.
+"""
+# pylint: disable=invalid-name
+
+import numpy as np
+
+
+def cos(
```

Comment:
An idea: could rename this file to`fit_function` instead of `fit_functions` and do a module import so these functions are accessed like `fit_function.cos` `fit_function.gaussian`.